### ISSUE TITLE
[python] Unregistered type QPSolverOutput

### ISSUE THREAD

**Issue by stephane-caron** [Simple-Robotics/proxsuite#7]
Trying to get ``results.status.info`` from an example from the docs yields:

```
TypeError: Unregistered type : proxsuite::proxqp::QPSolverOutput

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "test_prox.py", line 40, in <module>
    print(results.info.status)
TypeError: Unable to convert function return value to a Python type! The signature was
	(self: proxsuite.proxsuite_pywrap_avx2.proxqp.Info) -> proxsuite::proxqp::QPSolverOutput
```

Here is the example from the docs with the extra print instruction:

```python
import proxsuite
import numpy as np
import scipy.sparse as spa

def generate_mixed_qp(n, seed=1):
    # A function for generating random convex Qps

    np.random.seed(seed)
    n_eq = int(n / 4)
    n_in = int(n / 4)
    m = n_eq + n_in

    P = spa.random(
        n, n, density=0.075, data_rvs=np.random.randn, format="csc"
    ).toarray()
    P = (P + P.T) / 2.0

    s = max(np.absolute(np.linalg.eigvals(P)))
    P += (abs(s) + 1e-02) * spa.eye(n)
    P = spa.coo_matrix(P)
    q = np.random.randn(n)
    A = spa.random(m, n, density=0.15, data_rvs=np.random.randn, format="csc")
    v = np.random.randn(n)  # Fictitious solution
    delta = np.random.rand(m)  # To get inequality
    u = A @ v
    l = -1.0e20 * np.ones(m)

    return P, q, A[:n_eq, :], u[:n_eq], A[n_in:, :], u[n_in:], l[n_in:]


# load a Qp object using Qp problem dimensions
n = 10
n_eq = 2
n_in = 2
H, g, A, b, C, u, l = generate_mixed_qp(n)

# solve the problem using the sparse backend
results = proxsuite.proxqp.sparse.solve(H, g, A, b, C, u, l)
print(results.info.status)
```

**Comment by jcarpent** [comment]
@Bambade could you check and fix it?

**Comment by jcarpent** [comment]
Solved with #11
